addons/resource.language.en_gb/resources/strings.po

```diff
@@ -6208,7 +6208,7 @@ msgstr ""
 
 #: system/settings/settings.xml
 msgctxt "#13435"
-msgid "Enable HQ Scalers for scalings above"
+msgid "Enable HQ scalers for scaling above"
```

Comment:
There are >= three hq scalers: lanzcos 3 optimized, lanczos 2, bicubic, ..

plural is fine. If you want to make it mathematically correct: Only apply one of the _eventually_ selected hq scaling algorithms, when scaling above :-)